### DISCUSSION TITLE
Manual Unlock Request Page

### DISCUSSION THREAD

**Discussion by SporyTike** [RetroAchievements/RAWeb#1476] [Ideas]
I would like to have a page on the website where users can request manual unlocks as the official way to go. Our community grows and so does the amount of requests. Many of them don't even follow the requirements like ticket ID, achievement ID and valid proof. I would like to get some page like a form where players can input their information and it can only be submitted when everything is filled correctly.
Since it would require a decent amount of work to make a whole manual unlock system with request IDs and such which is not really doable until the whole website change I thought a workaround would be to use the input and send a formatted message to a site account extra created for manual unlocks.

I created some images to show how I imagine it.

**Concept for form design on the website:**
![image](https://user-images.githubusercontent.com/45672834/232245215-2e9c69a2-501c-41ac-a532-524ea993a1e9.png)
All fields and multi-choice require input and will throw a warning if you press submit with them being empty or containing characters that are not allowed e.g. you input letters in the achievement / ticket ID.

**Inbox Message sent to the site account:**
![image](https://user-images.githubusercontent.com/45672834/232246241-c8419420-bab4-4c34-8174-73c5f3bc2154.png)
The Achievement ID section will show the IDs twice, once as text (easy copy-paste in the unlock tool) and once with achievement link references [ach=] for easy investigation.
Ticket ID only requires the ticket link references [ticket=] for easy accessibility and confirming the ticket is filled correctly.
The Link to Proof section can be any valid link and is not related to a retroachievement.org-link.

EDIT: After submitting this feature it came to my mind that some "Additional Info" section at the end of the form might be good too which would be a simple optional input field where users can add additional info about the request if needed. For example when they want to explain why the proof is valid like it showing content only unlocked when the achievement conditions were met, proof is from a subset and whatnot.

**Comment by Jamiras** [comment]
This information should be moved to #711
